### DISCUSSION TITLE
Airflow home changes permissions

### DISCUSSION THREAD

**Discussion by BretaGlac** [apache/airflow#38116] [General]
So we recently upgraded to 2.8.2 and started having issues with SSH operators:
paramiko.ssh_exception.AuthenticationException: Authentication failed.

After some investigations, we found that permission on home folder of our airflow user has been changed. Seems to me that the cause of all our troubles could be this #37310 

I think that it should stop changing permissions when it reaches home directory. Any thoughts?

**Comment by potiuk** [comment]
Yes. You are right, can you please open an issue for that?

In the meantime, changing `file_task_handler_new_folder_permissions` to 0x755 should fix your issue.
